File: Smodr/src/main/java/com/kevintcoughlin/smodr/SmodrApplication.java
package com.kevintcoughlin.smodr;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

/**
 * The Smodr {@link Application}.
 *
 * @author kevincoughlin
 */
public final class SmodrApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(this);
    }
}

File: Smodr/src/main/java/com/kevintcoughlin/smodr/SmodrApplication.kt
package com.kevintcoughlin.smodr

import android.app.Application

class SmodrApplication : Application() {

    lateinit var container: AppContainer
        private set

    override fun onCreate() {
        super.onCreate()
        CrashReporter.init(this)
        container = AppContainer(this)
    }
}